test/torch/tensors/test_additive_shared.py

```diff
@@ -860,6 +888,47 @@ def test_argmax(workers, protocol):
     assert (ids.long() == torch.argmax(t, dim=1)).all()
 
 
+@pytest.mark.parametrize("protocol", ["snn", "fss"])
+def test_argmin(workers, protocol):
+    me, alice, bob, crypto_provider = (
+        workers["me"],
+        workers["alice"],
+        workers["bob"],
+        workers["james"],
+    )
+
+    args = (alice, bob)
+    kwargs = {"protocol": protocol, "crypto_provider": crypto_provider}
+
+    t = torch.tensor([3, 1.0, 2])
+    x = t.fix_prec().share(*args, **kwargs)
+    print(x.argmin)
+    idx = x.argmin().get().float_prec()
+    assert idx == torch.tensor([1.0])
+
+    t = torch.tensor([3, 4.0])
+    x = t.fix_prec().share(*args, **kwargs)
+    idx = x.argmin().get().float_prec()
+    assert idx == torch.tensor([0.0])
+
+    t = torch.tensor([3, 4.0, 5, 2])
+    x = t.fix_prec().share(*args, **kwargs)
+    idx = x.argmin().get().float_prec()
+    assert idx == torch.tensor([3.0])
+
+    # no dim
+    t = torch.tensor([[1, 2.0, 4], [3, 9.0, 2.0]])
+    x = t.fix_prec().share(*args, **kwargs)
+    ids = x.argmin().get().float_prec()
+    assert ids.long() == torch.argmin(t)
+
+    # dim=1
+    t = torch.tensor([[1, 2.0, 4], [3, 1.0, 2.0]])
+    x = t.fix_prec().share(*args, **kwargs)
+    ids = x.argmin(dim=1).get().float_prec()
+    assert (ids.long() == torch.argmin(t, dim=1)).all()
```

Comment:
Q: Could you also add a test with one hot and keepdim?

Reply:
sure!

Reply:
I added it 😄